GameData/ROEngines/RealPlume/HG3_SSTUKatniss_RealPlume.cfg

```diff
@@ -0,0 +1,99 @@
+@PART[ROE-HG3]:NEEDS[RealPlume]:BEFORE[RealPlume]
+{
+	@MODULE[ModuleEngineConfigs]
+	{
+		%type = ModuleEnginesRF
+	}
+	
+	//main wispy red plume
+	PLUME
+	{
+		name = Cryogenic_LowerRed_CE
+		transformName = J-2-ThrustTransform   //HG-3 model, but transform names are unchanged
+		localRotation = 0,0,0
+		localPosition = 0,0,0
+
+		speed = 1
+		energy = 1
+		emissionMult = 1
+		alphaMult = 0.5
+
+		corePosition = 0,0,2.5
+		coreScale = 0.5
+
+		plumePosition = 0,0,1.5
+		plumeScale = 1.5
+
+		plume2Scale = 1.5
+	}
+	
+	//secondary white flare
+	PLUME
+	{
+		name = Hydrolox_UpperBlue
+		transformName = J-2-ThrustTransform
+		localRotation = 0,0,0
+		localPosition = 0,0,0
+		
+		energy = 1
+		speed = 1
+		emissionMult = 1
+
+		flarePosition = 0,0,1
+		flareScale = 0.4
+
+		fumePosition = 0,0,0
+		fumeScale = 0
+
+		streamPosition = 0,0,0
+		streamScale = 0
+
+		shockconePosition = 0,0,0
+		shockconeScale = 0
+
+	}
+}
+@PART[ROE-HG3]:FOR[zzPostRealPlumeROEngines]:NEEDS[SmokeScreen]
+{
+	@EFFECTS
+	{
+		//adjust plume so only flare is generated
+		@Hydrolox_UpperBlue
+		{
+			@MODEL_MULTI_SHURIKEN_PERSIST[Hydrolox_UpperBlue-flare]
+			{
+				%saturationMult = 0.1
+			}
+			@MODEL_MULTI_SHURIKEN_PERSIST[Hydrolox_UpperBlue-stream]
+			{
+				%emissionMult = 0.0
+			}
+			@MODEL_MULTI_SHURIKEN_PERSIST[Hydrolox_UpperBlue-shockcone]
+			{
+				%emissionMult = 0.0
+			}
+			@MODEL_MULTI_SHURIKEN_PERSIST[Hydrolox_UpperBlue-fume]
+			{
+				%emissionMult = 0.0
+			}
+			@AUDIO
+			{
+				@clip = NULL
+				@volume = 0
+			}
+		}
+		@Cryogenic_LowerRed_CE
+		{
+			|_ = CombinedPlume
+		}
+		@Hydrolox_UpperBlue
+		{
+			|_ = CombinedPlume
+		}
+	}
+	@MODULE[ModuleEngines*]
+	{
+		@name = ModuleEnginesRF
+		%powerEffectName = CombinedPlume
+	}
```

Comment:
Changing the ModuleEngines module type in a plume patch still seems strange.  Setting the powerEffectName is fine.

Add closing linebreak to last line

Reply:
Yeah, agree, but the same thing is going on in the J-2s plume, which I obviously haven't touched.